Components/ListSearchResults/Partials/Header/index.twig

```diff
@@ -6,7 +6,6 @@
     <div class="searchForm-field boxShadow">
       <div class="searchForm-box">
         <label class="searchForm-label" for="search-input">
```

Comment:
I think this (and the corresponding) CSS should be removed.

Reply:
I would advise against this, as it has negative impact on accessibility - screen readers in particular, as described in [this article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#Why_you_should_use_labels).

Would be great to hear if you agree with this approach.

Reply:
I'm totally in for labels, but in order to be accessible, they need to be visible and have text. I think the article doesn't say anything different. So to my understanding an empty invisible label doesn't add anything and should be removed. An arial-label could be introduced instead onto the input field, but I don't have enough knowledge and experience about that to make a positive decision.